apps/server/src/shared/domain/entity/course.entity.ts

```diff
@@ -83,6 +89,21 @@ export class Course extends BaseEntityWithTimestamps implements ILearnroom, IEnt
 		if (props.startDate) this.startDate = props.startDate;
 	}
 
+	private getStudentIds(): EntityId[] {
```

Comment:
why not making this public and get rid of `getUserIds` as this anyway just works with the Ids?

Reply:
This should be changed on all entitys and interfaces to getStudentIds(). It is in wip state. :D

Reply:
In current iteration it is renamed and public.